src/main/resources/application.properties

```diff
@@ -1,4 +1,8 @@
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.username=utopia
-spring.datasource.password=NAg6K0V2guDxAXQC
-spring.datasource.url=jdbc:mysql://localhost:3306/utopia
+spring.datasource.username=root
+spring.datasource.password=Password
+spring.datasource.url=jdbc:mysql://localhost:3306/utopia_airlines
+
+
+spring.jpa.hibernate.ddl-auto = update
+
```

Comment:
Please, please don't version the application.properties file if you can help it - we're going to end up conflicting here because everyone's database credentials are different.